src/client/components/pages/parts/editor-achievements.js

```diff
@@ -125,19 +125,11 @@ class EditorAchievementTab extends React.Component {
 						<DragAndDrop name="rank3"/>
 					</div>
 					<div className="form-group">
```

Comment:
I've had a good result by removing  the `row` and `form-group` classes  from the preceding div (keeping only `dnd-container`), and deleting this div (moving the contents [the `<button>` and `<p>`] to be direct children of the `<form>`.

If you then add `padding-bottom-1` to the containing `<form>` element, the result is looking quite good !

Reply:
Thanks a lot! I tried these, and it is indeed good.
Will surely add in a few hours